config/rbac/kustomization.yaml

```diff
@@ -11,6 +11,9 @@ resources:
 - builder_role_binding.yaml
 - leader_election_role.yaml
 - leader_election_role_binding.yaml
+# Uncomment the following 2 lines if you want to deploy on OpenShift
```

Comment:
add this information on the README so users are aware of it.

Reply:
This still holds. Is it "resolved" now and these comments are valid or are you going to create the script you mentioned?